k-distribution/include/builtin/domains.k

```diff
@@ -1187,21 +1187,21 @@ module STDOUT-STREAM
 
 endmodule
 
-module MINT
-  imports INT
-  imports K-EQUAL
-  imports LIST
-
-  syntax MInt [hook(MINT.MInt)]
-
+module MINT-SYNTAX
   /*@\section{Description} The MInt implements machine integers of arbitrary
    * bit width represented in 2's complement. */
-
+  syntax {Width} MInt{Width} [hook(MINT.MInt)]
```

Comment:
This parameterization should be fine for the Haskell backend, but what will `Width` be instantiated with in practice?